mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/SingleMailboxReindexingTask.java

```diff
@@ -32,14 +32,17 @@
 import org.apache.james.task.TaskExecutionDetails;
 import org.apache.james.task.TaskType;
 
+import com.google.common.annotations.VisibleForTesting;
+
 public class SingleMailboxReindexingTask implements Task {
 
     public static final TaskType MAILBOX_RE_INDEXING = TaskType.of("mailbox-reindexing");
 
     public static class AdditionalInformation extends ReprocessingContextInformation {
         private final MailboxId mailboxId;
 
-        AdditionalInformation(MailboxId mailboxId, int successfullyReprocessedMailCount,
+        @VisibleForTesting
+        public AdditionalInformation(MailboxId mailboxId, int successfullyReprocessedMailCount,
```

Comment:
`@VisibleForTesting` alone is not enough?

Reply:
I need access in webadmin dto serialization tests